packages/providers/src/utils/change-case/change-case.spec.ts

```diff
@@ -0,0 +1,559 @@
+import {
+  capitalCase,
+  constantCase,
+  dotCase,
+  kebabCase,
+  noCase,
+  pascalCase,
+  pathCase,
+  sentenceCase,
+  snakeCase,
+  trainCase,
+} from './index';
+const studPrimitiveObject = {
```

Comment:
I think this is a typo?

```suggestion
const stubPrimitiveObject = {
```